Task6/script.js

```diff
@@ -0,0 +1,33 @@
+const app = document.getElementById("js-app");
+const fragment = document.createDocumentFragment();
```

Comment:
IMO
Place it as close to the calling side as possible.

Reply:
I placed getElement and Fragment, which were specified in the first line, right before forEach.